core/domain/stats_domain.py

```diff
@@ -1379,23 +1522,26 @@ class AnswerCalculationOutput:
     calculation.
     """
 
-    def __init__(self, calculation_output_type):
+    def __init__(self, calculation_output_type: object):
```

Comment:
Can the type be more specific than "object"?

Reply:
Done